HW7/task1.py

```diff
@@ -0,0 +1,44 @@
+# 1. Створити телефонну книгу, яка матиме наступні команди:
+# stats: кількість записів; add: додати запис; delete <name>: видалити запис за іменем (ключем)
+# list: список всіх імен в книзі; show <name>: детальна інформація по імені
+
+print("Hello! I am a simple phone book. My commands:\nstats, add, delete, list, show")
+
+phone_book = { "Adam" : 56474,
+               "Andreo" : 45453,
+               "Melisa" : 78475,
+}
+while True:
+    command = input("Enter command: ").lower()
+
+    if command == "stats":
+        print(f"I have {len(phone_book)} numbers")
+
+    if command == "add":
+        name = input("Name: ")
+        if name in phone_book:
+            print(f"You can't add {name}, it already exists")
+            continue
+        number = input("Number: ")
+        phone_book = {**phone_book, name: int(number)}
```

Comment:
nit: I think it is not required to unpack all data of the phone book here, new data can be assigned like `phone_book[name] = number`
and phone number could be in string format rather than int